lib/differ.rb

```diff
@@ -0,0 +1,63 @@
+module Differ
+  class << self
+    def get_dmp_diffs before, after
+      dmp = DiffMatchPatch.new
+      # https://github.com/google/diff-match-patch/wiki/API#diff_maintext1-text2--diffs
+      diffs = dmp.diff_main(before, after)
+      # https://github.com/google/diff-match-patch/wiki/API#diff_cleanupsemanticdiffs--null
+      dmp.diff_cleanupSemantic(diffs)
+      diffs
+    end
+
+    def calculate_ranges dmp_diffs
+      dmp_diffs.reduce([]) do |diffs, diff|
```

Comment:
Could one of these variables ( diff or diff ) be renamed to make it less confusing?